Kudu.FunctionalTests/Jobs/WebJobsTests.cs

```diff
@@ -12,12 +12,15 @@
 using Kudu.Client.Infrastructure;
```

Comment:
I think you should get unit test coverage complete for all the code you're about to check in, before you check in, so we don't have test debt right out of the gate.